### PR TITLE
Arnasv/api

### PR DESCRIPTION
**Pull request by 4rgetlahm**

### CODE REVIEW

File: GameLibrary/Database/DataContext.cs [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+				"server=" + Configuration.GetInstance().Settings["server"]
+				+ ";database=" + Configuration.GetInstance().Settings["database"]
+				+ ";user=" + Configuration.GetInstance().Settings["user"]
+				+ ";password=" + Configuration.GetInstance().Settings["password"]
+				+ ";port=" + Configuration.GetInstance().Settings["port"]
+			);
+		}
+
+		protected override void OnModelCreating(ModelBuilder modelBuilder)
+        {
```
Comment (martynasslav) [inline GameLibrary/Database/DataContext.cs:35]: Weird spacing

File: GameLibrary/Inventory/Item.cs [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+			this.Characters = characters;
+		}
+
+		protected Item ()
+		{
+
+		}
+
+		public int GetStats()
+        {
```
Comment (martynasslav) [inline GameLibrary/Inventory/Item.cs:49]: Weird spacing

File: GameLibrary/Location/Location.cs [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+		public Location (string DisplayName, LocationType LocationType, double Latitude, double Longtitude, int Radius)
+		{
+			this.DisplayName = DisplayName;
+			this.LocationType = LocationType;
+			Coordinate = new Coordinate(Latitude, Longtitude);
+			this.Radius = Radius;
+		}
+
+		protected Location()
+        {
```
Comment (martynasslav) [inline GameLibrary/Location/Location.cs:90]: Weird spacing

**Review by eimanteras — APPROVED** [review]

File: GameLibrary/GameLibrary.csproj [hunk 4]
```diff
@@ -0,0 +1,15 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <OutputPath>C:\Users\Arnas\source\repos\object-group-game\Server\lib\</OutputPath>
```
Comment (martynasslav) [inline GameLibrary/GameLibrary.csproj:8]: <OutputPath>$(SolutionDir)\Server\lib\</OutputPath>

Comment (4rgetlahm) [reply GameLibrary/GameLibrary.csproj:8]: This was due to Visual Studio's setting, changed it to this

File: GameLibrary/Database/DataContext.cs (hunk 1 above)
Comment (4rgetlahm) [reply GameLibrary/Database/DataContext.cs:35]: Spacing problems might be due to GitHub, as they're not present on Editor, if the issues occur on your end, please change it accordingly

**Pull request merged**